app/src/test/java/com/eveningoutpost/dexdrip/Models/BlueReaderTest.java

```diff
@@ -344,6 +346,7 @@ public void initialize() {
         // :: Verify
         assertThat(getLogs()).contains("I/blueReader: initialize blueReader!");
         assertThat(Pref.getInt("bridge_battery", -1)).isEqualTo(0);
+        assertThat(PersistentStore.getDouble("blueReaderFirmwareValue") == 0d ) ;
```

Comment:
try assertThat(PersistentStore.getDouble("blueReaderFirmwareValue").isWithin(0.0d).of(0d);